app/client/apps/goat/app/[lng]/(dashboard)/content/ContentManagement.tsx

```diff
@@ -22,7 +22,7 @@ import {
 import { formatDate } from "@/lib/utils/helpers";
 import FolderIcon from "@mui/icons-material/Folder";
```

Comment:
Can we also change these ones to FA (inc. ArrowDropDownIcon, StarIcon)

Reply:
done